kubernetes/spec/support/e2e/executor.rb

```
@@ -0,0 +1,511 @@
+          "coverage" => coverage_reporter.payload
+        }
+
+        failed = result.fetch("coverage").fetch("summary").fetch("failed")
+        raise ExecutionError.new("real API execution had #{failed} failures", result) if failed.positive?
+
+        result
+      end
+
+      private
+
+      def with_kubeconfig(cluster_manager)
```

RuboCop (1.85.1)

[convention] Use 2 (not 0) spaces for indented_internal_methods indentation.

(Layout/IndentationWidth)